SpatialGDK/Source/SpatialGDK/Public/Interop/SpatialDispatcher.h

```diff
@@ -31,10 +30,38 @@ class SPATIALGDK_API USpatialDispatcher : public UObject
 	void Init(USpatialNetDriver* NetDriver);
 	void ProcessOps(Worker_OpList* OpList);
 
+	// AddOpCallback returns a callback ID which is incremented on each callback that is registered.
+	// ComponentId must be in the range 1000 - 2000.
+	// Callbacks can be deregistered through passing the corresponding callback ID to the RemoveOpCallback function.
+	template<typename T>
+	using UserCallback = TFunction<void(T)>;
```

Comment:
I think I'm against calling anything User{Something}. If it's in a public API then a user will just think of it as a callback rather than a UserCallback.

Similarly I just noticed we never actually call this with a free parameter - we always know what the full concrete type is whenever we use it. So we could define AddComponentCallback etc.
Not sure if we want to but it might make it more obvious to a user what is expected. I don't have a strong opinion on this.

Reply:
Sounds like the reasonable conclusion reached is to just delete is because it creates more ambiguity than it solves?

Reply:
It is more verbose then but it might be cleaner. Your call. If there is no alias then it isn't a breaking change for us to make one.